### DISCUSSION TITLE
rAudio 1 • 20220317: Playlist window doesn't highlight active track anymore

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#856] [Fixed Issues]
No additional Text

**Comment by AndyFx12** [comment]
Still there in today´s update 20220327.

**Reply by rern** [reply]
Let me see: `status.sh | jq`

**Reply by AndyFx12** [reply]
{
  "player": "mpd",
  "btclient": false,
  "consume": false,
  "control": "Digital",
  "counts": {
    "album": 836,
    "albumartist": 205,
    "artist": 3568,
    "composer": 110,
    "conductor": 4,
    "date": 99,
    "genre": 153,
    "playlists": 1,
    "latest": 1,
    "nas": 1,
    "sd": 0,
    "usb": 0,
    "song": 13455,
    "webradio": 84
  },
  "icon": "",
  "librandom": false,
  "relays": false,
  "relayson": false,
  "scrobble": false,
  "stoptimer": false,
  "stream": false,
  "updateaddons": false,
  "updating_db": false,
  "volume": 98,
  "volumemute": 0,
  "webradio": false,
  "repeat": false,
  "random": false,
  "single": false,
  "file": "NAS/QNAP/Charts/The Dome 54/Opposite Of Adults.mp3",
  "playlistlength": 5,
  "song": 3,
  "state": "stop",
  "timestamp": 1648363195734,
  "Album": "The Dome Vol. 54",
  "Artist": "Various Artists",
  "Time": 193,
  "Title": "Opposite Of Adults",
  "ext": "MP3",
  "coverart": "/mnt/MPD/NAS/QNAP/Charts/The Dome 54/cover.1648065329.jpg",
  "sampling": "4/5 &bull; 44.1 kHz 160 kbit/s &bull; MP3",
  "elapsed": 0
}

**Reply by klassikmann** [reply]
Same here. Playlist windows starts ok, but after switching to playback window and proceeding to next track the playlist window is stuck.

rAudio:/srv/http/bash  #  ./status.sh | jq
{
  "player": "mpd",
  "btclient": false,
  "consume": false,
  "control": "",
  "counts": {
    "album": 3035,
    "albumartist": 815,
    "artist": 903,
    "composer": 401,
    "conductor": 0,
    "date": 82,
    "genre": 19,
    "playlists": 0,
    "latest": 3,
    "nas": 1,
    "sd": 0,
    "usb": 0,
    "song": 20123,
    "webradio": 115
  },
  "icon": "",
  "librandom": false,
  "relays": false,
  "relayson": false,
  "scrobble": false,
  "stoptimer": false,
  "stream": false,
  "updateaddons": false,
  "updating_db": false,
  "volume": 100,
  "volumemute": 0,
  "webradio": false,
  "repeat": false,
  "random": false,
  "single": false,
  "file": "NAS/Musik/Klassik+Romantik/Beethoven, Ludwig van/Sinfonien/Jordi Savall + Le Concert des Nations/Sinfonie No 1 C-Dur, Op. 21/02. II. Andante cantabile con moto.flac",
  "playlistlength": 4,
  "song": 1,
  "state": "play",
  "timestamp": 1648363634047,
  "Album": "Ludwig van Beethoven: Sinfonie No 1 C-Dur, Op. 21",
  "Artist": "Jordi Savall + Le Concert des Nations",
  "Time": 413,
  "Title": "II. Andante cantabile con moto",
  "ext": "FLAC",
  "coverart": "/mnt/MPD/NAS/Musik/Klassik+Romantik/Beethoven, Ludwig van/Sinfonien/Jordi Savall + Le Concert des Nations/Sinfonie No 1 C-Dur, Op. 21/folder.1648363386.jpg",
  "sampling": "2/4 &bull; 24 bit 96.0 kHz 2.59 Mbit/s &bull; FLAC",
  "elapsed": 267
}

**Comment by rern** [comment]
@klassikmann and @AndyFx12,

Please try the UPDATE branch.

**Comment by AndyFx12** [comment]
Hi rern,
issue is fixed. Thanks!


State of active playlist-item should be refreshed, even if playlist-tab is not active (see main.js, $( '.btn-cmd' ).click( function() ):
![grafik](https://user-images.githubusercontent.com/35779271/160277321-e36b4ae1-bfe9-4db5-bef2-f49d3feace6e.png)

otherwise, non active-items could show still elapsed-time and playback-symbol.

**Reply by klassikmann** [reply]
Only partly fixed. After switching to playback window and proceeding to next track the playlist window looks like:
<img src="https://user-images.githubusercontent.com/25383513/160277742-e89366da-4d09-4781-9399-ce598ad05261.jpg" width=600>
See the former track.

**Reply by klassikmann** [reply]
@rern 
Observation:
The UPDATEd assets/js/function.js seems to fix the problem.

**Comment by AndyFx12** [comment]
Fix confirmed - non-active items are refreshed fine, now.
